dapp/src/components/Banner.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react'
+import classnames from 'classnames'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { useStoreState } from 'pullstate'
+import { fbt } from 'fbt-runtime'
+import moment from 'moment'
+import StakeStore from 'stores/StakeStore'
+import { adjustLinkHref } from 'utils/utils'
+
+const Banner = ({ dapp }) => {
+  const burn = moment('2022-10-10T00:00:00.000Z')
```

Comment:
probably, can move it outside of this component's scope. If it's a constant, we don't have to recompute it every time the component rerenders. Not a big deal though, since it won't make much difference